assets/scripts/library-api.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-module.exports = true;
+const getBooksAuthor = function() {
+  return $.ajax({
+    url: 'http://localhost:3000/books/',
+    method: 'GET',
+  });
+};
+
+const getBooksTitle = function() {
```

Comment:
`getBooksAuthor`, `getBooksTitle`, and `getAllBooks` are all the same function, so you could have just re-used it. If this were a real application, this sort of duplication would be OK so long as the plan were to make the functions differ once API functionality was provided for these kinds of searches.